### ISSUE TITLE
Intermittent UI test failure - < CustomTabsTest.customTabCopyToolbarUrlTest>

### ISSUE THREAD

**Issue by AndiAJ** [mozilla-mobile/fenix#24416]
### Stacktrace:
[Logcat](https://github.com/mozilla-mobile/fenix/files/8334422/customTabCopyToolbarUrlTest.txt)

**Comment by AndiAJ** [comment]
Flaky 1x on 3/23 Main [Firebase link](https://console.firebase.google.com/u/0/project/moz-fenix/testlab/histories/bh.66b7091e15d53d45/matrices/7681397052728475747/executions/bs.9abcd2fcda9cae2f/test-cases)

**Comment by sv-ohorvath** [comment]
A11y issue causing the failure was fixed, can be reopened if it fails again.
